File: 5 Exercises/jsexamples/sumFibs.js
function sumFibs(num) {
	if (num < 0) {
		return undefined;
	}
	var sumOdds = 0;
	for (var n1 = 0, n2 = 1, temp; n2 <= num; temp = n2, n2 += n1, n1 = temp) {
		sumOdds += n2 * (n2 % 2);
	}
	return sumOdds;
}

File: 5 Exercises/sumFibs.c
#include <cs50.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int sumFibs(int n)
{
    int sum = 1, prevFib = 1, newFib = 1;

    do
    {
        if (newFib % 2 != 0)
        {
            sum += newFib;
        }
        int temp = prevFib;
        prevFib = newFib;
        newFib += temp;
    } while (newFib <= n);
    return sum;
}



int main( void)
{
    printf("%s\n", "Enter (n) to continue: ");
    int n = get_int();
    int o = sumFibs(n);
    printf("%i\n", o);
    return 0;
}